explorer-ui/src/components/Testimonials/Testimonials.jsx

```diff
@@ -1,13 +1,60 @@
 import * as React from "react"
 import "./Testimonials.css"
-import {
-    Link
-} from "react-router-dom";
+const colors = ["#000000", "#808080", "#000000"];
+const delay = 2500;
 
 export default function Testimonials() {
-    return (
-        <div className="testimonials">
-            
-        </div>
-    )
+  const [index, setIndex] = React.useState(0);
+  const timeoutRef = React.useRef(null);
+
+  function resetTimeout() {
+    if (timeoutRef.current) {
+      clearTimeout(timeoutRef.current);
+    }
+  }
+
+  React.useEffect(() => {
+    resetTimeout();
+    timeoutRef.current = setTimeout(
+      () =>
+        setIndex((prevIndex) =>
+          prevIndex === colors.length - 1 ? 0 : prevIndex + 1
+        ),
+      delay
+    );
+
+    return () => {
+      resetTimeout();
+    };
+  }, [index]);
+
+  return (
+    <div className="slideshow">
+      <div
+        className="slideshowSlider"
+        style={{ transform: `translate3d(${-index * 100}%, 0, 0)` }}
+      >
+        {colors.map((backgroundColor, index) => (
+          <div
+            className="slide"
+            key={index}
+            style={{ backgroundColor }}
+          ></div>
+        ))}
+      </div>
+
+      <div className="slideshowDots">
+        {colors.map((_, idx) => (
```

Comment:
not using the values ?

Reply:
No, i'm using the background color to set the style in the mapping call before this but the purpose of this mapping call is to keep track of the dots corresponding to each slide.